stripe/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -189,7 +190,7 @@ internal class DefaultFlowController internal constructor(
         if (paymentSelection == PaymentSelection.GooglePay) {
             googlePayLauncher(
                 StripeGooglePayContract.Args(
-                    paymentIntent = initData.paymentIntent,
+                    paymentIntent = initData.stripeIntent as PaymentIntent,
```

Comment:
Can we explicitly `error()` if `stripeIntent` is not a `PaymentIntent`? It might help for future debugging.